docs/3.0.0-alpha.x/getting-started/quick-start-tutorial.md

```diff
@@ -141,21 +135,13 @@ After your admin user is registered, you will see the Strapi admin panel:
 
 ![Strapi Admin Panel](../assets/quick-start-detailed/AfterRegistrationScreenAdminPanel.png 'Strapi Admin Panel')
 
----
-
-Every user is found by clicking in the left menu: **CONTENT TYPES** --> **Users**.
-
-From here you can add and edit users.
-
-![Strapi Users Page](../assets/quick-start-detailed/AdminUserInUsers.png 'Strapi Users Page')
-
 ## 4. Create a Content Type
 
 At this point, your project is empty. You need to create a **Content Type**.
 
-We can define a **Content Type** as a collection of entered data represented by fields.  For example, a **Content Type** called `restaurant` may be intended to display information regarding restaurants. A `restaurant` has a `name`, a main `image`, a `description`, maybe even a link to the `restaurant menu`.
+We can define a **Content Type** as a collection of entered data represented by fields. For example, a **Content Type** called `restaurant` may be intended to display information regarding restaurants. A `restaurant` has a `name`, a main `image`, a `description`, maybe even a link to the `restaurant menu`.
```

Comment:
@soupette - the original blog post and article use image, and the videos use image.  I can see your point of view, but changing it here would mean changing it in a million other places and it's not that significant.  When we redo it, I promise to use "cover"...

Reply:
@davidkartuzinski here's my thinking : we have developed an example app [foodadvisor](https://github.com/strapi/foodadvisor/blob/master/api/api/restaurant/models/Restaurant.settings.json) which uses a `cover` attribute instead of an `image` one and I think that even though this app is not integrated in our documentation we should have some consistencies in both our repositories and our documentation.

Reply:
@soupette - I totally understand.  But we haven't even begun to incorporate foodadvisor into our docs and there are four videos in the wild, 6 blog posts (strapi blog, gatsby blog, hackernoon, etc) that would all have to be updated.

I actually 100% agree with you and in my own blog it is called a "header" image (which is the most common way to call this).  But until we are really going to go to foodadvisor, I don't really see that we need to change this and create an inconsistency between everything else already published.  

For example, a **header image** is defined in this blog post https://www.pexels.com/blog/header-image-inspiration/ and a **cover image** is different, a cover image refers to images used in social media profiles as background - https://www.peopleperhour.com/blog/blogroll/the-pph-cover-image-guide/ 

At some point, I will have to look more closely at Foodadvisor and things like this will have to be clarified, but for now, I still think we should leave as is.